File: Music/Shared/Models/Album.cs
using Music.Shared.Cores;

namespace Music.Shared.Models;

public class Album : QQMusic
{
    public string? SongId { get; init; }

    public override string GetImageUrl(Image image)
        => $"https://y.qq.com/music/photo_new/T002R{ (int)image }x{ (int)image }M000{ Id }.jpg";
}

File: Music.Client.Android/Kotlin/Music/lib/src/main/java/com/zitherharp/music/model/Album.kt
package com.zitherharp.music.model

import com.zitherharp.music.core.QQMusic

class Album(id: String): QQMusic(id) {

    override fun getImageUrl(image: Image) =
        "https://y.qq.com/music/photo_new/T002R${image.pixel}x${image.pixel}M000$id.jpg"
}